Library/Homebrew/caveats.rb

```diff
@@ -46,6 +47,14 @@ def zsh_completion_caveats
     end
   end
 
+  def fish_completion_caveats
+    if keg and keg.completion_installed? :fish and which("fish") then <<-EOS.undent
```

Comment:
But the other completion functions also use `and`? That would be inconsistent then?